engine/table/src/main/java/io/deephaven/engine/table/impl/select/MatchFilter.java

```diff
@@ -88,11 +84,13 @@ public enum CaseSensitivity {
         MatchCase, IgnoreCase
     }
 
+    // TODO NATE NOCOMMIT: USE BUILDER TO CREATE MATCH FILTER??
+
```

Comment:
Be sure to delete. Let's defer introducing a builder.

However, I think it might be time to add JavaDoc to the constructors:
"This constructor is called by X, for Y."
Or turn them into static methods with meaningful names.

```suggestion
```